source/features/remove-zero-packages.tsx

```diff
@@ -0,0 +1,22 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {getRepoURL} from '../libs/utils';
+
+function init(): void {
+	const packageElem: HTMLElement | null = select(`.numbers-summary a[href='/${getRepoURL()}/packages']`);
+	if (packageElem && packageElem.innerText.trim() === '0 packages') {
```

Comment:
```suggestion
	if (packageElem && packageElem.textContent.trim() === '0 packages') {
```

Use `textContent` instead.

Reply:
In this case it doesn't work as expected. I sent another suggestion

<img width="186" alt="" src="https://user-images.githubusercontent.com/1402241/62419981-3752ff00-b6b6-11e9-80a8-30050e285eb0.png">

Reply:
Disregard this, if possible try targeting `.num` inside the anchor which only contains the package count.

Reply:
That's what I thought, but my suggestion with `startsWith` works